doc/arbiter.md

```diff
@@ -7,3 +7,8 @@ ROHD HCL implements a generic `abstract` [`Arbiter`](https://intel.github.io/roh
 The [`PriorityArbiter`](https://intel.github.io/rohd-hcl/rohd_hcl/PriorityArbiter-class.html) is a combinational (stateless) arbiter that always grants to the lowest-indexed request.
 
 [PriorityArbiter Schematic](https://intel.github.io/rohd-hcl/PriorityArbiter.html)
+
+## Round Robin Arbiter
+
+The Roun Robin Arbiter is a sequential (stateless) arbiter which grants requests same ways as Priority Arbiter, but keeps track of last request granted.
+The arbiter uses two masks to determine the next request to be attended. 
```

Comment:
The masks are an implementation detail and not relevant to the user, we can remove this.